platform/fabric/core/generic/channel.go

```diff
@@ -172,6 +177,27 @@ func newChannel(network *network, name string, quiet bool) (*channel, error) {
 	return c, nil
 }
 
+var defaultCacheSize = 100
```

Comment:
shall we have this variable included in the already existing var block at the top of the file?

Reply:
moved it to vault